ADA/databricks_workflow_script_databricks.qmd

```diff
@@ -0,0 +1,247 @@
+---
+title: "Scripting Workflows in Databricks"
```

Comment:
This is an alternative to the point and click method of creating a workflow, right? Might be worth adding a quick sentence to remind people of that at the top of the page

Reply:
Have added:

Workflows can be constructed through the Databricks Workflows user interface (UI), however for large or complex workflows the UI can be a time consuming way to build a workflow. In these scenarios it is quicker and more inline with RAP principles to script your workflow.